cmd/cosign/cli/verify/verify.go

```diff
@@ -177,29 +179,59 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 		}
 	}
 	if keylessVerification(c.KeyRef, c.Sk) {
```

Comment:
Is there a test we could add to e2e_test.go to exercise this new code?

Reply:
will look into this shortly

Reply:
still TODO

Reply:
added functional tests in https://github.com/sigstore/cosign/pull/3464/commits/a25df3b890a02c1c2e0f081ec220120b211c1aef (as discussed, generating certificates on the fly using https://github.com/sigstore/cosign/blob/main/test/cert_utils.go). @haydentherapper

Reply:
@haydentherapper added functional tests in https://github.com/sigstore/cosign/pull/3464/commits/bd10952e2ecabf0f2ad20c1de98f0887c87a91b2